packages/replay/src/replay.ts

```diff
@@ -1100,6 +1101,23 @@ export class ReplayContainer implements ReplayContainerInterface {
       return;
     }
 
+    const start = this._context.initialTimestamp;
```

Comment:
I wonder if this is reliable to use as the start of the session? We should trace where this and the session start gets updated

Reply:
we cannot use `session.start` here as that may be outdated for e.g. buffer session etc. What we care about (?? I think at least?) is the first event that may actually be sent, which should be this timestamp?